_includes/footer.html

```diff
@@ -1,5 +1,6 @@
 {% assign t = site.data.locales[page.lang][page.lang] %}
-<button id="scrollToTopButton" title="Go to top">Scroll to Top</button>
+
+<button id="scrollToTop" title="Go to top">Scroll to Top</button>
```

Comment:
Why did you change the ID attribute from "scrollToTopButton" to "scrollToTop"? Because of this change it does not work as it should.